### ISSUE TITLE
Minning notes not spawning in Steamvault 

### ISSUE THREAD

**Issue by mollyklich** [SunstriderEmu/BugTracker#2286]
There seems to be missing Mining notes in Steamvault. There are no rich adamantite notes, and atleast two missing nodes, that are always there, are missing. 

Furthermore there are usually gasclouds inside SV. Have not seen a single one in over 50+ runs.

**Comment by SUN-Leekie** [comment]
Can you please provide us the locations?

**Comment by mollyklich** [comment]
yea sure. I've linked a picture downbelow (not from endless tho - hope it's okay, or I will fix a new)

So it's to the right from where you go upstairs to the next floor. Behind the boglords. I've posted a picture more with the more precise location. This is a minning spot that should always be there! And it was there on endless untill lately.

![x](https://user-images.githubusercontent.com/62138428/76677479-02554600-65cf-11ea-9a90-1e333ce126d9.jpg)

![z](https://user-images.githubusercontent.com/62138428/76677556-f1f19b00-65cf-11ea-87dc-85aaccf8176b.png)

Tell me if u need anymore information.
